packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -32,21 +32,29 @@ export default function Container() {
   return <Kubes {...state} />;
 }
 
+const docUrl =
+  'https://goteleport.com/docs/kubernetes-access/guides/multiple-clusters/';
+
 export function Kubes(props: State) {
-  const { kubes, attempt } = props;
+  const { kubes, attempt, user, isEnterprise, isLeafCluster } = props;
 
   return (
     <FeatureBox>
       <FeatureHeader alignItems="center" justifyContent="space-between">
         <FeatureHeaderTitle>Kubernetes</FeatureHeaderTitle>
+        {isEnterprise && isLeafCluster ? null : (
+          <ButtonPrimary as="a" target="_blank" href={docUrl}>
+            View documentation
+          </ButtonPrimary>
+        )}
```

Comment:
this logic doesn't handle the second condition:
```
- do not show a button if user is using Enterprise and:
  - user is in leaf cluster or
  - user can't create
 ```

Also I would read up on conditional rendering: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

And short circuit conditionals: https://codeburst.io/javascript-short-circuit-conditionals-bbc13ac3e9eb#:~:text=Short%20circuiting%20means%20that%20in,look%20at%20the%20second%20operand.

Reply:
Good to know. I've moved it to a new component and handled the logic there.